packages/fortranxml/package.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Fortranxml(AutotoolsPackage):
+    """Fortran XML (FoX) s a library, written entirely in Fortran, designed to allow
+    the easy use of XML from Fortran programs. FoX caters for both XML output
+    and input.
+    """
+
+    homepage = "https://github.com/andreww/fox"
+    git = "https://github.com/andreww/fox.git"
+
+    maintainers = ["andreww"]
+
+    version("4.1.2", sha256="dea0adc9cc035238fa9cdba42f2bf56481e3a64ac8aa0aece9119f127f71d4e7")
+
+    flag_handler = AutotoolsPackage.build_system_flags
+
+    def url_for_version(self, version):
+        return "https://github.com/andreww/fox/tarball/{0}".format(version)
+
+    def install(self, spec, prefix):
+        super(AutotoolsPackage, self).install(spec, prefix)
```

Comment:
Hi @micaeljtoliveira , I haven't seen this before in an SPD. Should it be `super().install(spec, prefix)`? This might be something we should do in our existing SPDs, so let's have a chat about it next week.